.github/workflows/pyupgrade.yml

```
@@ -9,9 +9,14 @@ jobs:
 
     steps:
       - uses: actions/checkout@v4
-      - name: Install uv
-        uses: astral-sh/setup-uv@v3
       - uses: actions/setup-python@v5
```

Is there any reason to not specify `python-version` here?
```suggestion
      - uses: actions/setup-python@v5
        with:
          python-version: '3.x' 
```

---

> Is there any reason to not specify `python-version` here?

Probably because `uvx` makes a virtualenv, and Ubuntu's system Python protection doesn't kick in?